src/auth/oauth/errors.rs

```diff
@@ -63,7 +63,7 @@ impl actix_web::ResponseError for OAuthError {
             | OAuthErrorType::ScopesTooBroad
             | OAuthErrorType::AccessDenied => {
                 if self.valid_redirect_uri.is_some() {
-                    StatusCode::FOUND
+                    StatusCode::OK
```

Comment:
What is the issue that is causing this to be needed? I remember Jackson was being particular about the error codes, is this fixing it to adhere to the oauth spec or was oauth spec not working with knossos?